slam_toolbox/src/merge_maps_kinematic.cpp

```diff
@@ -248,8 +251,24 @@ bool MergeMapsKinematic::mergeMapCallback(
       tf2::Transform submap_correction = submap_marker_transform_[id];
       transformScan(iter, submap_correction);
       transformed_scans.push_back((*iter));
+
+      if (iter == it_LRV->begin() && id != 1){
```

Comment:
Please add a space between `) {`. Also you didn't indent your if/else code. Finally, remove all spaces between pointer accessors.